BrowserKit/Sources/TabDataStore/TabFileManager.swift

```diff
@@ -144,7 +144,7 @@ public struct DefaultTabFileManager: TabFileManager {
             return windowData
         } catch {
             logger.log("Error decoding window data: \(error)",
-                       level: .debug,
+                       level: .warning,
```

Comment:
@OrlaM Just bumped this from debug to warning for parsing errors (hopefully ok).